src/components/Controls/Input/index.tsx

```diff
@@ -6,10 +6,12 @@ const Input: FunctionComponent<
 		borders: "bottom" | "all"
 		number?: boolean
 		validation?: string | null
+		staticPlaceholder?: string
 	} & Omit<InputHTMLAttributes<HTMLInputElement>, "type">
-> = ({borders, number = false, validation, ...inputProps}) => {
+> = ({borders, number = false, validation, staticPlaceholder, ...inputProps}) => {
```

Comment:
`staticPlaceholder` might be not the best naming for this prop, maybe something like `prefix` would work better... 
But for me it sounds as most clear